mintapi/signIn.py

```diff
@@ -335,7 +335,11 @@ def sign_in(
     driver.implicitly_wait(20)  # seconds
     driver.get(url)
     if not beta:
-        element = driver.find_element_by_link_text("Sign in")
+        element = WebDriverWait(driver, 20).until(
```

Comment:
@CalvinChanCan If I'm not mistaken, WebDriverWait doesn't actually return the element but I think a bool value. I'd put the wait above the original find element and see if that works.

Reply:
@burkematthew From the Selenium [docs](https://www.selenium.dev/documentation/webdriver/waits/#:~:text=el%20%3D%20WebDriverWait(driver%2C%20timeout%3D3).until(lambda%20d%3A%20d.find_element_by_tag_name(%22p%22))), WebDriverWait can return the element if it is found and be interpreted as a truthy bool value. Though I have tried as suggested and still its still erroring on the click but does proceed to the sign in page though 😕.

Reply:
@CalvinChanCan interestingly enough (though I couldn't really explain why), if you move the assignment of element and the click to its own method, say, home_page, and put the WebDriverWait before you call home_page, it seems to work as expected:

```
WebDriverWait...
home_page(driver)
```

Reply:
@burkematthew Do you mean how I have it in the latest commit? For me, I'm still experiencing the same error still.

Reply:
@CalvinChanCan that's super weird as it resolved the issue for me. 

I guess the time.sleep will have to suffice then. Thanks for trying out the other options!

By the way, go ahead and just leave the home_page method in place. A little cleaner anyways!

Reply:
@burkematthew, yeah it is strange how its resolving for some users and not for others. Though I have observed that sometimes mint inconsistently routing me to the new sign in page UI or to the old sign in page UI. The sign in page UI is completely new to me and I have never seen it until when this issue started occurring so I wonder if its related. 

![image](https://user-images.githubusercontent.com/13265811/175829798-956551ee-ad18-4a26-b3d0-fe97f9277442.png)
![image](https://user-images.githubusercontent.com/13265811/175829838-b96ed645-ce84-446e-8991-e4015e177d9b.png)

Thanks again! I have kept the home_page method and added back `time.sleep`.

Reply:
@CalvinChanCan you can take out the WebDriverWait since we're doing the explicit sleep.

Reply:
@burkematthew Done!